src/server.c

```diff
@@ -2048,6 +2048,10 @@ int main (int argc, char **argv) {
     }
   #endif
 
+  #ifdef DEBUG_WOLFSSL
```

Comment:
This belongs in mod_openssl.c in the mod_openssl_init() function, which is called when the module is loaded.  The call does not belong in server.c